pkg/config/setup/system_probe.go

```diff
@@ -209,6 +210,7 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Config) {
 	cfg.BindEnvAndSetDefault(join(spNS, "enable_conntrack_all_namespaces"), true, "DD_SYSTEM_PROBE_ENABLE_CONNTRACK_ALL_NAMESPACES")
 	cfg.BindEnvAndSetDefault(join(netNS, "enable_protocol_classification"), true, "DD_ENABLE_PROTOCOL_CLASSIFICATION")
 	cfg.BindEnvAndSetDefault(join(netNS, "enable_ringbuffers"), true, "DD_SYSTEM_PROBE_NETWORK_ENABLE_RINGBUFFERS")
+	cfg.BindEnvAndSetDefault(join(netNS, "enable_kernel_batching"), false, "DD_SYSTEM_PROBE_NETWORK_ENABLE_KERNEL_BATCHING")
```

Comment:
I settled on `enable_custom_batching`. Let me know if that works?

Reply:
I suppose it's not a big difference, but why not `enable_in_buffer_batching`? That is how we've referred to the feature/proposal internally

Reply:
That would do the opposite of this config. This config is enabling the "old" behavior, with the expectation that it will go away once/if you folks are happy with in-buffer batching.

Reply:
ah right. In that case I am happy with `enable_custom_batching` 👍